src/Surfnet/ServiceProviderDashboard/Application/Service/ServiceStatusService.php

```diff
@@ -92,4 +92,44 @@ public function getEntityStatus(Service $service)
         // No published or drafted entities discovered, state "No"
         return Service::ENTITY_PUBLISHED_NO;
     }
+
+
+    /**
+     * - Status: "No" when no test entity, and no draft on test is present
```

Comment:
Please update the doc block. It speaks of the test env. Where the prod env is applicable.

Reply:
Done, also renamed the method to `getConnectionStatus` to reflect the purpose of the method better.